src/app/public/theming/theme-if.directive.spec.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  ComponentFixture,
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  expect
+} from '@skyux-sdk/testing';
+
+import {
+  SkyThemeModule
+} from '../theme.module';
+
+import {
+  SkyThemeIfTestComponent
+} from './fixtures/theme-if-test.component';
+
+import {
+  SkyThemeSettings
+} from './theme-settings';
+
+import {
+  SkyTheme
+} from './theme';
+
+import {
+  SkyThemeMode
+} from './theme-mode';
+
+import {
+  SkyThemeService
+} from './theme.service';
+
+describe('ThemeIf directive', () => {
+  let fixture: ComponentFixture<SkyThemeIfTestComponent>;
+  const defaultThemeSettings = new SkyThemeSettings(SkyTheme.presets.default, SkyThemeMode.presets.light);
+  const modernThemeSettings = new SkyThemeSettings(SkyTheme.presets.modern, SkyThemeMode.presets.dark);
+  const themeService = new SkyThemeService();
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        SkyThemeIfTestComponent
+      ],
+      imports: [
+        SkyThemeModule
+      ],
+      providers: [
+        { provide: SkyThemeService, useValue: themeService }
+      ]
+    });
+    TestBed.inject(SkyThemeService);
```

Comment:
The way your code currently is, there's no need to call `TestBed.inject()` here unless you're trying to get the instance of `SkyThemeService` from the `TestBed`. My suggestion is to not construct `SkyThemeService` yourself but let `TestBed` do it for you, then call `TestBed.inject()` to get the instance.

```
TestBed.configureTestingModule({
  ...
  providers: [
    SkyThemeService
  ]
});

themeService = TestBed.inject(SkyThemeService);
```